### PR TITLE
login with steam button added

### PR DESCRIPTION
**Pull request by hmankodiya**
Issue:
A new button idea of adding "Login with  Steam" button in the social buttons type implemented .
image

### CODE REVIEW

File: assets/js/buttons-examples.js [hunk 1]
```diff
@@ -524,6 +524,10 @@ $(document).ready(function () {
           classes: "discord",
           text: "Login with Discord",
         },
+        {
```
Comment (shahednasser) [inline assets/js/buttons-examples.js:527]: Please place this in the correct alphabetical order under "Social"

Comment (hmankodiya) [reply assets/js/buttons-examples.js:527]: sure I'll re-arrange it.

Comment (hmankodiya) [reply assets/js/buttons-examples.js]: Issue Resolved.

**Comment by shahednasser** [comment]
Great work thank you!

@all-contributors please add @hmankodiya for code and idea

**Pull request merged**